### ISSUE TITLE
Wild reporting with pokestop screenshot

### ISSUE THREAD

**Issue by TheGerontocracy** [PokeNavBot/issue-tracker#276]
Wild reporting (together with nests) would nicely round off the capabilities of the bot. 

Given the great technology that's available for quests, and how similiar it could be implemented, I think it's a pretty low hanging fruit.

**Suggested implementation**
Implement more or less as quests: 

1) Players upload a screenshot of a pokestop together with the name of the pokemon in a designated channel (announce-wild-here). Additional information can be added after a comma. Examples: 
* *Chansey*, Screenshot-of-pokestop
* *Chansey, 100 IV*, Screenshot-of-pokestop.
The bot reads those messages and responds with a message with embeds similar to that for quests.

2) After 1 hour the bot deletes it's own messages. 

This feature request is inspired by a suggestion by CaptnAwesome187 in feature-requests on the discord.

**Comment by Hype365** [comment]
I completely agree with this suggestion :) Currently our server uses Meowth for wilds reporting but we don't use the roles system from it, we use pretty much exactly like above, just to announce where a wild is sighted. We would ditch Meowth completely in favor of using exclusively PokeNav :) 👍

**Comment by BHollowell** [comment]
+1 from me on this. New to PokeNav and had some members asking if this functionality was available. It really would give PokeNav just about everything you could ask for.

**Comment by MagicSword89** [comment]
We absolutely need this, with rocket pokestop takeovers being a thing soon. Pokenav will be the one bot stop for any pogo server

**Comment by AsphyxiAsian** [comment]
I searched for this on the server with many hits, so I'm sad it's not being looked at more. Our server is full of collectors. Their previous "want" feature allowed them to get notified when something they wanted was available as research, reported in a raid, reported in the wild, or someone had it up for trade. I understand the arguments in regards to roles, but we are a smaller server and we've always made it work. I'm sure there is a way to make this work. We don't even need the screenshot function for this.
